flow/src/main/scala/org/alephium/flow/core/BlockFlowGroupView.scala

```diff
@@ -252,13 +252,6 @@ object BlockFlowGroupView {
       mempool: MemPool
   ) extends Impl0[WS](worldState, blockCaches) {
 
-    override def getPreContractOutput(
-        outputRef: ContractOutputRef
-    ): IOResult[Option[ContractOutput]] = {
-      // MemPool ignores ContractOutputs
-      super.getPreContractOutput(outputRef)
-    }
```

Comment:
This is removed now.

Reply:
Agree :+1: for me it was 50/50, I decided to make it more explicit